### DISCUSSION TITLE
Direct link to a draft item

### DISCUSSION THREAD

**Discussion by kortsmit** [community/community#19782] [Projects and Issues] [answered]
My team is very much enjoying the Projects beta so far and excited to see what future enhancements will be rolled out.

One feature that we've missed is the ability to directly link to a draft item (where the dialog panel with details is already open) without needing to first convert it to a full issue within a repository. 

We've gotten around this so far by applying a filter and sharing that link, but it would be great if the drafts each had their own URL where we could share a link after putting a draft together.

**Comment by kortsmit** [comment, marked as answer]
I just ran into a reference to direct/deep linking to drafts here with a response. My apologies, wasn't able to find anything yesterday!

https://github.com/github-community/community/discussions/15350#discussioncomment-2702024

**Comment by mattcosta7** [comment]
https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/

We recently shipped urls that link directly to the projects sidepanel open to an issue or draft issue
